src/main/java/stsjorbsmod/actions/ExhaustNegativeCardAction.java

```diff
@@ -0,0 +1,42 @@
+package stsjorbsmod.actions;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardGroup;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+
+import java.util.ArrayList;
+
+// Based on ExhaustSpecificCardAction
+public class ExhaustNegativeCardAction extends AbstractGameAction {
+    private CardGroup pile;
+
+    public ExhaustNegativeCardAction(AbstractCreature source, CardGroup pile) {
```

Comment:
I was initially confused by the word "Negative" here, I read it as the "with a value < 0" form rather than the "with a deterimental effect" form. Maybe it would be less ambiguous to instead use `ExhaustCardsMatchingPredicateAction` and have `IvoryTower` pass a `Predicate<AbstractCard>` of `c -> c.type == CardType.CURSE || c.type == CardType.STATUS`?

Reply:
Great idea; updated.